Compile/interface/Producers/LeptonSFProducer.h

```diff
@@ -16,192 +18,75 @@ class LeptonSFProducer : public ZJetProducerBase
                  ZJetSettings const& settings) const override;
 
   protected:
-    float m_sf[30][30] = {};
-    float m_er[30][30] = {};
-    float m_sfdata[30][30] = {};
-    float m_erdata[30][30] = {};
-    float m_sfmc[30][30] = {};
-    float m_ermc[30][30] = {};
-    std::vector<float> m_xbins = {};
-    std::vector<float> m_ybins = {};
-    std::vector<float>* m_etabins = {};
-    std::vector<float>* m_ptbins = {};
     std::string m_sffile;
-    std::string m_id;
     std::string histoname;
-    std::vector<int> runs;
-    bool m_etarev2D[3] = {};
+    std::string weightName;
     bool m_reversed_axes;
     bool m_absoluteeta;
```

Comment:
I know, we don't have a consistent naming convention and this was already beforem but may change the variable name to `m_absoluteEta`?

Reply:
Ok